packages/react-native/ReactCommon/jsc/CMakeLists.txt

```diff
@@ -10,6 +10,15 @@
 cmake_minimum_required(VERSION 3.13)
 set(CMAKE_VERBOSE_MAKEFILE on)
 
+# Copy jsc headers into JavaScriptCore subdirectory
+get_target_property(JSC_PREFAB_INCLUDE_DIR jsc INTERFACE_INCLUDE_DIRECTORIES)
+set(JSC_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/include)
+file(MAKE_DIRECTORY ${JSC_INCLUDE_DIR})
+file(GLOB JSC_HEADERS "${JSC_PREFAB_INCLUDE_DIR}/*.h")
+foreach(HEADER ${JSC_HEADERS})
+        file(COPY ${HEADER} DESTINATION "${JSC_INCLUDE_DIR}/JavaScriptCore")
```

Comment:
not a huge fan of doing this. I guess is fine for now, but I'd love for us to avoid doing io inside CMake files.

Couldn't we call the prefab 'JavaScriptCore' so we wouldn't have to do any copy and pasting?

Reply:
alright. let me add the JavaScriptCore prefix in jsc-android